src/PhpSpreadsheet/Calculation/Calculation.php

```diff
@@ -2185,7 +2185,7 @@ class Calculation
         ],
         'SERIESSUM' => [
             'category' => Category::CATEGORY_MATH_AND_TRIG,
-            'functionCall' => [MathTrig::class, 'SERIESSUM'],
+            'functionCall' => [MathTrig\SeriesSum::class, 'funcSeriesSum'],
```

Comment:
Not `evaluate()`?

Reply:
SeriesSum was not part of this PR. I just noticed that I had neglected to update the function table for SeriesSum when I did implement its replacement. Since I am planning a follow-up PR to change funcWhatever, I would prefer to defer this suggestion until then.